.gitignore

```diff
@@ -3,4 +3,5 @@
 Gemfile.lock
 pkg/*
 *.DS_Store
-.rvmrc
\ No newline at end of file
+spec/db/test.db
+.ruby-version
```

Comment:
Makes sense.  I was just curious because you locked the travis ruby version, while removing it locally.